packages/frontend/src/components/kpi-simulation-chart/index.tsx

```diff
@@ -145,8 +151,8 @@ export function SimulationChart({
 
     if (error) {
         return (
-            <div className={styles.root}>
-                <div className={styles.emptyContainer}>
+            <div className={`root ${styles.root} ${className}`}>
+                <div className={`emptyContainer ${styles.emptyContainer}`}>
```

Comment:
I think we can use `classNames` here?

Reply:
Addressed in 2f8d27fdcd5a228a9dd3b57f094e2adfd960ef24.